perllib/FixMyStreet/Cobrand/Smidsy.pm

```diff
@@ -64,12 +66,43 @@ sub get_severity {
         reverse @{ $self->severity_categories };
 }
 
+sub get_police_info {
+    my ($self, $report) = @_;
+
+    my ($lat, $lon) = ($report->latitude, $report->longitude);
+
+    my $ua = new LWP::UserAgent();
+    $ua->agent("collideoscope.org.uk");
+    my $r = $ua->get("https://data.police.uk/api/locate-neighbourhood?q=$lat,$lon");
+
+    my $j = JSON->new->utf8->allow_nonref;
+    try {
+        $j = $j->decode($r->content);
+    } catch {
+        return;
+    };
+    return unless $j->{force};
+
+    my @data = eval { FixMyStreet->path_to('../collideoscope/data/police-force-urls.csv')->slurp };
```

Comment:
Likewise, if this fails then no sense doing the network things so possibly check for this first?